conduit/Data/Conduit/Internal/Conduit.hs

```diff
@@ -842,6 +842,9 @@ yield :: Monad m
 yield o = yieldOr o (return ())
 {-# INLINE yield #-}
 
+-- | Send a monadic value to the next component to consume.
```

Comment:
I think it would be worth including the word "downstream," and explaining why this is different from `yield`.